packages/discord.js/src/structures/ThreadMember.js

```diff
@@ -101,11 +101,10 @@ class ThreadMember extends Base {
 
   /**
    * Removes this member from the thread.
-   * @param {string} [reason] Reason for removing the member
-   * @returns {ThreadMember}
+   * @returns {Promise<ThreadMember>}
```

Comment:
This could be merged immediately in a separate pull request without breaking changes. Do you mind doing that?

Reply:
Bien sûr.